control-plane/api-gateway/binding/binder_test.go

```diff
@@ -55,19 +55,19 @@ var (
 )
 
 type resourceMapResources struct {
-	grants                   []gwv1beta1.ReferenceGrant
-	secrets                  []corev1.Secret
-	gateways                 []gwv1beta1.Gateway
-	httpRoutes               []gwv1beta1.HTTPRoute
-	tcpRoutes                []gwv1alpha2.TCPRoute
-	meshServices             []v1alpha1.MeshService
-	services                 []types.NamespacedName
-	jwtProviders             []*v1alpha1.JWTProvider
-	gatewayPolicies          []*v1alpha1.GatewayPolicy
-	externalAuthFilters      []*v1alpha1.RouteAuthFilter
-	consulInlineCertificates []api.InlineCertificateConfigEntry
```

Comment:
should we have both here so that we don't miss any regressions on how we handle inline certs?

Reply:
So by default on upgrade the API Gateway will be replacing the inline cert with the file-system. So may not be an issue here?